lib/kb_uploadmethods/Utils/UnpackFileUtil.py

```diff
@@ -20,6 +20,27 @@ def log(message, prefix_newline=False):
 
 class UnpackFileUtil:
 
+    # staging file prefix
+    STAGING_GLOBAL_FILE_PREFIX = '/data/bulk/'
+    STAGING_USER_FILE_PREFIX = '/staging/'
+
+    def _get_staging_file_path(self, token_user, staging_file_subdir_path=''):
+        """
+        _get_staging_file_path: return staging area file path
+
+        directory pattern:
```

Comment:
What is the "directory pattern" ? not sure what that means
Typo in perfered

Reply:
👍 Thanks. Good catch.